github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -658,6 +658,47 @@ presubmits:
           resources:
             requests:
               memory: "4Gi"
+
+  - name: pull-kubevirt-gosec
+    cluster: ibm-prow-jobs
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    skip_report: true
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+    spec:
```

Comment:
please add

`preset-bazel-cache: "true"`

here, then you can remove the env vars as theya re all covered with the preset